<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>total_main</class>
 <widget class="QDialog" name="total_main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::TabFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(0, 0, 48);
</string>
  </property>
  <widget class="QLabel" name="logo">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>41</width>
     <height>54</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Vivaldi</family>
     <pointsize>26</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color:rgb(0, 0, 90);
border::rgb(0, 0, 90);</string>
   </property>
   <property name="text">
    <string> fg</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>41</width>
     <height>431</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color:rgb(0, 0, 90);
border:black;</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="Search_b">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
background-color:rgb(0, 0, 140);
border::rgb(0, 0, 140);</string>
    </property>
    <property name="text">
     <string>П</string>
    </property>
   </widget>
   <widget class="QPushButton" name="EXIT_B">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>55</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
background-color:rgb(0, 0, 140);
border::rgb(0, 0, 140);</string>
    </property>
    <property name="text">
     <string>В</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="contacts_logo">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>0</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color:rgb(0, 0, 90);
border::rgb(0, 0, 90);</string>
   </property>
   <property name="text">
    <string>              Контакти</string>
   </property>
  </widget>
  <widget class="QLabel" name="line">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>0</y>
     <width>21</width>
     <height>481</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color:rgb(0, 0, 90);
border::rgb(0, 0, 90);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="sms_text">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>400</y>
     <width>361</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color:rgb(0, 0, 140);
border::rgb(0, 0, 140);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Clear_b">
   <property name="geometry">
    <rect>
     <x>719</x>
     <y>420</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color:rgb(0, 0, 140);
border::rgb(0, 0, 140);</string>
   </property>
   <property name="text">
    <string>--&gt; X</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Send_b">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>450</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color:rgb(0, 0, 140);
border::rgb(0, 0, 140);</string>
   </property>
   <property name="text">
    <string>--&gt;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="ALL_LABEL">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>50</y>
     <width>441</width>
     <height>331</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color:rgb(0, 0, 140);
border::rgb(0, 0, 140);</string>
   </property>
  </widget>
  <widget class="QLabel" name="contacts_logo_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>0</y>
     <width>461</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color:rgb(0, 0, 90);
border::rgb(0, 0, 90);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Connect_B">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>10</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color:rgb(0, 0, 140);
border::rgb(0, 0, 140);</string>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Select_Cust">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>180</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color:rgb(0, 0, 140);
border::rgb(0, 0, 140);</string>
   </property>
   <property name="text">
    <string>Обрати Користувача</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="INFO_IN">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>261</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color:rgb(0, 0, 90);</string>
   </property>
   <property name="title">
    <string>Введіть дані:</string>
   </property>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>201</width>
      <height>26</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="name_in_L">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
background-color:rgb(0, 0, 140);</string>
     </property>
     <property name="text">
      <string>Ім'я:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="name_in">
     <property name="maximumSize">
      <size>
       <width>117</width>
       <height>26</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
background-color:rgb(0, 0, 140);


</string>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>201</width>
      <height>26</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="second_name_in_L">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
background-color:rgb(0, 0, 140);</string>
     </property>
     <property name="text">
      <string>Прізвище:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="second_name_in">
     <property name="maximumSize">
      <size>
       <width>117</width>
       <height>26</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
background-color:rgb(0, 0, 140);


</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>281</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color:rgb(0, 0, 90);
border::rgb(0, 0, 90);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="refresh_B">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>10</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color:rgb(0, 0, 140);
border::rgb(0, 0, 140);</string>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter_3">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>182</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="SEL_name">
    <property name="minimumSize">
     <size>
      <width>91</width>
      <height>21</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>91</width>
      <height>21</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
background-color:rgb(0, 0, 140);
border::rgb(0, 0, 140);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="SEL_sec_name">
    <property name="minimumSize">
     <size>
      <width>91</width>
      <height>21</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>91</width>
      <height>21</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
background-color:rgb(0, 0, 140);
border::rgb(0, 0, 140);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QTextBrowser" name="ALL_CUST">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>271</y>
     <width>261</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color:rgb(0, 0, 140);
border::rgb(0, 0, 140);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>460</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color:rgb(0, 0, 90);
border::rgb(0, 0, 90);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="KORUSTUBACHI">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>229</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color:rgb(0, 0, 140);
border::rgb(0, 0, 140);</string>
   </property>
   <property name="text">
    <string>              КОРИСТУВАЧІ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ID">
   <property name="geometry">
    <rect>
     <x>655</x>
     <y>10</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color:rgb(0, 0, 140);
border::rgb(0, 0, 140);
</string>
   </property>
   <property name="text">
    <string>Identyfy</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
